lambda-functions/event-receiver/app/makeregister_event_handler_test.go

```diff
@@ -78,7 +81,10 @@ func TestMakeRegisterEventHandler_Success(t *testing.T) {
 		user := args.Get(2).(*Actor)
 		*user = existingUser
 	}).Return(nil)
-	mockDynamo.On("Put", ctx, mock.Anything).Return(nil)
+	mockDynamo.On("Put", ctx, mock.Anything, mock.Anything).Return(nil)
```

Comment:
You should be able check the values passed to the Put function, though you'll need to add `mockFactory.On("UuidString").Return("mock-uuid")`
```suggestion
	mockDynamo.On("Put", ctx, "ActorUsers", map[string]types.AttributeValue{
		"Id":       &types.AttributeValueMemberS{Value: "mock-uuid"},
		"Identity": &types.AttributeValueMemberS{Value: "urn:fdc:gov.uk:2022:XXXX-XXXXXX"},
	}).Return(nil)
```